t/venom/vars/all.yml

```diff
@@ -501,3 +501,11 @@ inline_l2.node.ipaddress: 192.168.2.10
 inline_l2.pf_portal: 192.168.2.1
 inline_l2.networks: 192.168.2.0
 
+################################################################################
+#Security event autoreg test suite specific variables
+#################################################################################
+security_event_autoreg.node.mac: 00:03:00:11:11:01
+security_event_autoreg.node.mac_url_encoded: 00%3A03%3A00%3A11%3A11%3A01
```

Comment:
You can re-use existing variables: `node01_ens7_mac_address` and `node01_ens7_mac_address_url_encoded` with following syntax:

```yaml
security_event_autoreg.node.mac: '{{.node01_ens7_mac_address}}'
```